commands/management/add_external.go

```diff
@@ -158,4 +160,5 @@ func init() {
 		EnumVar(&AddExternal.MetricsMode, metricsModes...)
 	groupHelp := fmt.Sprintf("Group name of external service (default: %s)", defaultGroupExternalExporter)
 	AddExternalC.Flag("group", groupHelp).Default(defaultGroupExternalExporter).StringVar(&AddExternal.Group)
+	AddExternalC.Flag("skip-connection-checks", "Skip checks connection to exporter").BoolVar(&AddExternal.SkipConnectionChecks)
```

Comment:
```suggestion
	AddExternalC.Flag("skip-connection-check", "Skip checks connection to exporter").BoolVar(&AddExternal.SkipConnectionChecks)
```
Everywhere else we use `skip-connection-check` and in API too